react-counter/src/components/Counter.js

```diff
@@ -0,0 +1,30 @@
+import React, {Component} from 'react';
+import Plus from './Plus';
+import Minus from './Minus';
+import Refresh from './Refresh';
+import Display from './Display';
+
+class Counter extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {count: 0};
+  }
+
+  render() {
+    let {count} = this.state;
+    return (
+      <div className='main'>
+        <div className='display-wrapper'>
+          <Display count={count}/>
+        </div>
+        <div className='buttons-wrapper'>
+          <Plus onCountUp={() => {this.setState({ count: count + 1 })}}/>

```

Comment:
It is better to save the callback to the class instance.  The problem with this syntax is that a different callback is created each time the Counter renders. Read more about handling events https://reactjs.org/docs/handling-events.html